Engine/ResourceManagement/Importer/SkyboxImporter.cpp

```diff
@@ -0,0 +1,10 @@
+#include "SkyboxImporter.h"
+
+#include "Helper/Config.h"
+#include "Main/Application.h"
+#include "Module/ModuleFileSystem.h"
+
+FileData SkyboxImporter::ExtractData(Path& assets_file_path) const
+{
+	return assets_file_path.GetFile()->Load();
```

Comment:
is necessary a class just for one function?

Reply:
If at the future the skybox importing logic doesnt improve(like importing special skybox files) probably not.